fleet-management/src/main/java/cz/fi/muni/pa165/entity/Employee.java

```diff
@@ -19,6 +19,16 @@
     @Column(nullable=false)
     private String surname;
 
+    /**
+     * Initialize new Employee object. For serialization uses.
+     */
+    public Employee() {
```

Comment:
I just copy paste existing comment in one entity, I can remove it, but hibernate need no-arg constructor in entity ...